src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc

```diff
@@ -46,9 +49,48 @@ aws:
                 # true/false - ask AWS to automatically decrypt SecureString type with KMS
                 useSecureParameters: true
                 # String - you can use a custom root path the application will look for configurations in. The default is '/config/application'.
-                rootHierarchyPath: /config/application
+                rootHierarchyPath: /config
```

Comment:
Should the default be changed?

Reply:
what should it be changed to? The previous info was wrong. 'application' is put in there as default after the /config by Micronaut afaik.

Reply:
"you can use a custom root path the application will look for configurations in. The default is '/config/application'." So if that isn't the case then that statement should be updated